packages/cli/src/commands/database/seed/tables.ts

```diff
@@ -118,8 +119,9 @@ export const createTables = async (connection: DatabaseTransactionConnection) =>
   ];
   const sorted = allQueries.slice().sort(compareQuery);
   const database = await getDatabaseName(connection, true);
+  const password = generateStandardId(32);
```

Comment:
Is it a good practice to create a random string for the role password?
May be we need to get the password from the seed command.
@charIeszhao @gao-sun what do you think?

Reply:
Also, this feature better be optional.

Reply:
i think it's fine. this role is a base role with no login. the only concern is if we should store the password somewhere, or print it after seed

Reply:
Maybe we should add a new option in the db seed CLI command to generate this random password, and print it out in console if the command is executed with the option.

Reply:
Updated, now use `--encrypt-base-role` to create base role with password